build-common/NHibernate.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <PropertyGroup>
+    <VersionMajor Condition="'$(VersionMajor)' == ''">5</VersionMajor>
+    <VersionMinor Condition="'$(VersionMinor)' == ''">0</VersionMinor>
+    <VersionPatch Condition="'$(VersionPatch)' == ''">0</VersionPatch>
+    <VersionSuffix>Alpha1</VersionSuffix>
```

Comment:
This should be `<VersionSuffix Condition="'$(VersionSuffix)' == ''">Alpha1</VersionSuffix>` so it can be overwritten at build time?